### ISSUE TITLE
Non deterministic behavior

### ISSUE THREAD

**Issue by jschneider** [jmfayard/buildSrcVersions#95]
In my project the buildSrcVersions plugin toggles (sometimes) between these two deps:

`const val it_test_support: String = "org.neo4j.community:it-test-support:" + Versions.org_neo4j_community_it_test_support`
`const val org_neo4j_community_it_test_support: String = "org.neo4j.community:it-test-support:" +  Versions.it_test_support`

It is unclear for me why these changes happen. It does not change every time - but regularly.

**Comment by jmfayard** [comment]
@jschneider Sorry, I forgot to reply
I would be interested if you can find when this non-deterministic behavior happens.
But in the meantime, this should fix your problem

```
buildSrcVersions {
   // Configurations: see https://github.com/jmfayard/buildSrcVersions/issues/53

   useFqdn("it-test-support")
}
```

**Comment by jschneider** [comment]
I already have this configured. My configuration looks like that:

`
buildSrcVersions {
  rejectVersionIf {
    isNonStable(candidate.version) && !isNonStable(currentVersion)
  }

  useFqdnFor("mail", "com.intellij:annotations", "annotations", "it_test_support")
}
`

So maybe it is related to the configuration..... I think I will close this issue and wait until it happened again.

**Comment by jmfayard** [comment]
@jschneider I have reworked `useFqdnFor()` and I *may* have fixed the non-deterministic behavior. This is released in v0.6.5
